infra/modules/frontend/browser/README.md

```diff
@@ -0,0 +1,78 @@
+**copied from [https://github.com/chanzuckerberg/cztack/tree/v0.30.0/aws-single-page-static-site](https://github.com/chanzuckerberg/cztack/tree/v0.30.0/aws-single-page-static-site)**
```

Comment:
Is this a full copy? If so, maybe just linking to the up-to-date original would be sufficient

Reply:
Oh I see, it includes the `referer_secret`